<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qgis3DDockWidgetBase</class>
 <widget class="QDockWidget" name="qgis3DDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qgis3D</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="activate_3d">
      <property name="text">
       <string>Activate 3D</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QGroupBox" name="options_box">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>3D Options</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="3">
        <widget class="QToolButton" name="sync_now">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="view-refresh"/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="sync_box">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bidirectional</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Canvas -&gt; 3D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D -&gt; Canvas</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="sync_label">
         <property name="text">
          <string>Synchronization</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
